SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+## Supported Versions
+
+Please see [Releases](https://github.com/handshake-org/hsd/releases).
+We recommend using the [most recently released version](https://github.com/handshake-org/hsd/releases/latest).
+
+## Reporting a Vulnerability
+
+To report security issues send an email to chjjeffrey@gmail.com (not for support).
+
+The following keys may be used to communicate sensitive information to developers:
+
+| Name | Fingerprint | Full Key |
+|------|-------------|--------|
+| Christopher Jeffrey (@chjj) | B4B1F62DBAC084E333F3A04A8962AB9DE6666BBD | https://keybase.io/chjj |
+
+
+You can also import a key by running the following command with an individual’s fingerprint: `gpg --recv-keys "<fingerprint>"`
```

Comment:
When I run this command, it does not work:

```
$ gpg --recv-keys B4B1F62DBAC084E333F3A04A8962AB9DE6666BBD
```

Note that I do not have any special configuration with the `gpg`, its a rather fresh install. I'm not sure what key servers it is on besides keybase, maybe the command to import from keybase is included as well as the exact `gpg` command needed to import. I believe there is a flag to specify the keyserver to use.

Reply:
While my gpg could import it as it is (think it's modified config), searching for it on https://pgp.mit.edu/, http://keys.gnupg.net/, https://keyserver.ubuntu.com/ all returned Not Found until `0x` was prefixed to it. Maybe this hex prefix could be added if someone searches the key id on these sites.

Reply:
Does this command work for you both?

```
$ gpg --keyserver hkps://keyserver.ubuntu.com --recv-keys B4B1F62DBAC084E333F3A04A8962AB9DE6666BBD
gpg: key 8962AB9DE6666BBD: public key "Christopher Jeffrey <chjjeffrey@gmail.com>" imported
gpg: no ultimately trusted keys found
gpg: Total number processed: 1
gpg:               imported: 1
```

While we're at we can check mine as well:

```
$ gpg --keyserver hkps://keyserver.ubuntu.com --recv-keys E61773CD6E01040E2F1BD78CE7E2984B6289C93A
gpg: key E7E2984B6289C93A: public key "Matthew Zipkin (GitHub Signing Key) <pinheadmz@gmail.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

Reply:
Both work :+1:.